pkg/apis/apps/v1beta2/conversion.go

```diff
@@ -361,3 +368,89 @@ func Convert_extensions_RollingUpdateDeployment_To_v1beta2_RollingUpdateDeployme
 	}
 	return nil
 }
+
+func Convert_extensions_DaemonSet_To_v1beta2_DaemonSet(in *extensions.DaemonSet, out *appsv1beta2.DaemonSet, s conversion.Scope) error {
+	out.ObjectMeta = in.ObjectMeta
+	if out.Annotations == nil {
+		out.Annotations = make(map[string]string)
+	}
+	out.Annotations[appsv1beta2.DeprecatedTemplateGeneration] = strconv.FormatInt(in.Spec.TemplateGeneration, 10)
+	if err := Convert_extensions_DaemonSetSpec_To_v1beta2_DaemonSetSpec(&in.Spec, &out.Spec, s); err != nil {
+		return err
+	}
+	if err := s.Convert(&in.Status, &out.Status, 0); err != nil {
```

Comment:
@caesarxuchao @janetkuo It works. After adding that, `Convert_v1beta2_DaemonSetStatus_To_extensions_DaemonSetStatus` and `Convert_extensions_DaemonSetStatus_To_v1beta2_DaemonSetStatus` are auto-generated. I would like to add this in another PR. WDYT?